mongodbatlas/resource_mongodbatlas_global_cluster_config_test.go

```diff
@@ -143,7 +145,7 @@ func testAccCheckMongoDBAtlasGlobalClusterImportStateIDFunc(resourceName string)
 func testAccCheckMongoDBAtlasGlobalClusterAttributes(globalCluster *matlas.GlobalCluster, managedNamespacesCount int) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		if len(globalCluster.ManagedNamespaces) != managedNamespacesCount {
-			return fmt.Errorf("bad managed namespaces: %s", globalCluster.ManagedNamespaces)
+			return fmt.Errorf("bad managed namespaces: %v", globalCluster.ManagedNamespaces)
```

Comment:
`badly managed namespaces`

Reply:
Is `ManagedNamespaces` a struct?

Reply:
is a struct slice, the lint returned me this error:

``` 
printf: fmt.Errorf format %s has arg globalCluster.ManagedNamespaces of wrong type []go.mongodb.org/atlas/mongodbatlas.ManagedNamespace
```